csv_template_ingester/template_csv_parser.py

```diff
@@ -233,10 +233,10 @@ def add_fields(keywords, names, units, systs, sys_dict, row):
         elif 'datatype' == normalize(keywords[j]):
             systm = add_datatype(systm, cell)
 
-        elif 'preparationstepname' == normalize(keywords[j]) or 'processstepname' == normalize(keywords[j]):
+        elif 'preparationstepname' == normalize(keywords[j]) or 'processstepname' == normalize(keywords[j]) or 'preparationname' == normalize(keywords[j]) or 'processname' == normalize(keywords[j]) or 'prepname' == normalize(keywords[j]) or 'procname' == normalize(keywords[j]):
```

Comment:
This could be more elegantly phrased as:
if normalize(keywords[j]) in ['preparationstepname', 'processstepname', etc etc]: